### PR TITLE
release 1.1.1

### PR DESCRIPTION
**Pull request by JaredBoone**

### CODE REVIEW

**Comment by JaredBoone** [comment]
This is published

File: src/Button.jsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ export default class Button extends Component {
     this.state = {
       hoverSize: void 0,
       hover: false,
-      pressed: false
+      pressed: props.pressed
```
Comment (isuttell) [inline src/Button.jsx]: I'm going to be annoying here. Having state and props being called the same thing is an anti-pattern in my opinion. It gets confusing when reading the code to determine how/why the value changes over time. Can we rename the key of prop or the state?

Comment (JaredBoone) [reply src/Button.jsx]: I agree. Do you have a suggestion for an alternate name?

Comment (isuttell) [reply src/Button.jsx]: I often prefix the prop something like `initialPress` but we might want to consider just calling the prop `value` which would bring it in line with [standard React forms](https://reactjs.org/docs/forms.html)

Comment (JaredBoone) [reply src/Button.jsx]: OK, I've created issue https://github.com/ship-components/ship-components-buttons/issues/15 to address this. 1.1.1 is already published, I believe that change will need to go into a future release (eg 1.1.2).

**Pull request merged**